chapters/inheritance.tex

```diff
@@ -1149,7 +1149,8 @@ \subsection{Annotations for Redeclaration and Modification}\label{annotation-cho
 parameter Boolean useHeatPort = false annotation(choices(checkBox = true));
 \end{lstlisting}
 \end{example}
-\section{Selective Model Extension}\label{selective-model-extension}
+
+\section{Selective Model Extension}\label{selective-model-extension}\index{break@\robustinline{break}!deselection}\index{deselection}
```

Comment:
Two things.  First explain in the document index that the _deselection_ entry is really meant to bring you to a section that has nothing about deselection in the title:
```suggestion
\section{Selective Model Extension}\label{selective-model-extension}\index{deselection!selective model extension}
```

Second, move the other index entry to where the use of the keyword `break` is introduced further down in this section:
```
\begin{lstlisting}[language=modelica]
model C
  extends B($\ldots$, break $D$, $\ldots$);
  $\ldots$
end C;
\end{lstlisting}%
\index{break@\robustinline{break}!deselection}
```